test/dom-helpers.spec.ts

```diff
@@ -18,66 +18,111 @@
  * limitations under the License.
  */
 
-import { isSubmitButton, getSelector } from '../src/injected/dom-helpers';
+import { readFileSync } from "fs";
```

Comment:
Seems this file has been reformatted to use double quotes, unlike the other files in this repo — is there some misconfiguration here?

Reply:
yeah I'm not sure what happened here. Thanks!